packages/medical-domain/src/domains/repositories/statistics/insulin-statistics.service.ts

```diff
@@ -64,40 +66,76 @@ function getWeight(allPumpSettings: PumpSettings[]): ParameterConfig | undefined
   return weight ?? undefined
 }
 
-function getBasalBolusData(basalsData: Basal[], bolus: Bolus[], numDays: number, dateFilter: DateFilter): BasalBolusStatistics {
+function getBasalBolusData(basalsData: Basal[], bolus: Bolus[], mealBoluses: Wizard[], numDays: number, dateFilter: DateFilter): BasalBolusStatistics {
   if (numDays === 0) {
     return {
-      bolus: 0,
-      basal: 0,
-      total: 0
+      bolus:0,
```

Comment:
<!-- Nitpicks are trivial preference-based requests. These should be non-blocking by nature.  -->
**🤓 nitpick:** ‏Missing spaces here after `:` 

```suggestion
      bolus: 0,
```